src/main/resources/application.yaml

```diff
@@ -15,6 +15,17 @@ bullet:
     config: pubsub_defaults.yaml
     # The amount of time to sleep in ms between empty responses reads from the PubSub. Avoids busy waiting
     sleep-ms: 1
+  websocket:
+    # The prefix of server destination. It's used for client to send message to server(broker).
+    server-destination-prefix: /app
```

Comment:
Can we use dot separated variable names for our variables? I know spring uses dashes for its own.

Reply:
Done.